editoast/src/views/rolling_stocks.rs

```diff
@@ -39,12 +39,15 @@ pub enum RollingStockError {
     #[error("Name '{name}' already used")]
     #[editoast_error(status = 400)]
     NameAlreadyUsed { name: String },
+    #[error("RollingStock '{name}' is locked")]
+    #[editoast_error(status = 400)]
+    RollingStockIsLocked { name: String },
```

Comment:
We should rather return the id of the rolling stock than its name.

```suggestion
    RollingStockIsLocked { rolling_stock_id: i64 },
```